blueocean-dashboard/src/main/js/components/PipelinePage.jsx

```diff
@@ -116,4 +118,4 @@ PipelinePage.childContextTypes = {
 const selectors = createSelector([pipelineSelector],
     (pipeline) => ({ pipeline }));
 
-export default connect(selectors, actions)(documentTitle(PipelinePage));
+export default translate(['jenkins.plugins.blueocean.dashboard.Messages'], { wait: true })(connect(selectors, actions)(documentTitle(PipelinePage)));
```

Comment:
the export is the whole component.

Regarding the react-i18next there are benefits in using it since it will store the keys in a store and takes care of (um)mounting the desired languages and ns bundles.

I left an example in the bo-core-js https://github.com/jenkinsci/blueocean-plugin/pull/556/files#diff-edeae55984cff08bd75ec3c753690ff7R73 I agree that is very clean but those components are not supporting the react cycle as https://github.com/i18next/react-i18next/blob/master/src/translate.js